cloudinary-image-management-and-manipulation-in-the-cloud-cdn/js/src/components/featured-image.js

```diff
@@ -20,9 +20,19 @@ let FeaturedTransformationsToggle = ( props ) => {
 };
 
 // Setup our properties.
-FeaturedTransformationsToggle = withSelect( ( select, ownProps ) => ( {
-    overwrite_featured_transformations: select( 'core/editor' ).getEditedPostAttribute( 'meta' )[ '_cloudinary_featured_overwrite' ] ?? false,
-} ) )( FeaturedTransformationsToggle );
+FeaturedTransformationsToggle = withSelect( ( select, ownProps ) => {
+    let editor = select( 'core/editor' );
+
+    if ( editor.getEditedPostAttribute( 'meta' ) ) {
+        return {
+            overwrite_featured_transformations: editor.getEditedPostAttribute( 'meta' )[ '_cloudinary_featured_overwrite' ] ?? false,
+        };
+    }
+
+    return {
```

Comment:
@pereirinha my issue with this is that this disables this functionality completely.